test/test_operations.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2009-present MongoDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test the operations module."""
+
+from test import UnitTest
+from pymongo import ASCENDING, DESCENDING
+from pymongo.operations import IndexModel
+from pymongo.collation import Collation
+
+
+class TestOperations(UnitTest):
+    """Test the operations module.""" 
```

Comment:
Do you think the idea for a separate test class for each class is no good? It looks funny to have `"""Test the operations module."""` at the module-level and then to repeat it here. I would rather use the names `TestIndexModel` and `test_repr`, otherwise I think this is fine.